types/types.proto

```diff
@@ -207,6 +207,7 @@ message Header {
 	bytes data_hash = 7;
 	bytes validators_hash = 8;
 	bytes app_hash = 9;
+        bytes proposer = 10;
```

Comment:
It is a bit problematic what to actually put in this field:

* A public key? If so, we lose information about key type.
* An address? It encompasses both key type and the key itself, but they are encoded in a non-reversible hash.